translations/messages-en.properties

```diff
@@ -577,6 +577,7 @@ debug.db_info.title = Database Info
 debug.mode.title = Enable debug mode
 debug.mode.description = Debug mode will print information into the browser console to help developers diagnose problems with the application.  After changing this setting reload the app so it applies.
 android_app.version.title = Android App Version
+ddoc.version = DDoc Version
```

Comment:
"Application Version" may be confounded with the "webapp version", or the "android app version". Does something like "Config version" work?